pom.xml

```diff
@@ -171,6 +170,30 @@
         <version>1.2</version>
       </plugin>
     </plugins>
+    <pluginManagement>
+      <plugins>
```

Comment:
Maybe. I merely corrected the Maven warnings turned up by `clean install`. There might be other warnings displayed during `release:{prepare,perform}` that I did not see yet; AFAIK there is no good way to preview that (one of the reasons `maven-release-plugin` is a dumb design).